### PR TITLE
Polish and edit the readme

### PR DESCRIPTION
**Pull request by ghost**
to: @lannka, @rudygalfi
cc: @cramforce

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,78 +1,76 @@
...
 #### Cookie usage
-The library uses cookie `AMP_TOKEN` to store information. It serves 2 purposes:
-1. To persist a security token received from the API server which is to be used for exchanging CID next time.
-2. To act as a lock so that no concurrent requests being sent. 
 
-## Run test
+The  Google AMP Client ID library uses the `AMP_TOKEN` cookie to store information. This cookie serves two purposes:
+
+1. To persist a security token that is received from the API server, which can be used to exchange CID values next time.
+2.  To act as a lock so that no concurrent requests are being sent.
```
Comment (rudygalfi) [inline README.md]: nit: extra space before "To"

**Review by rudygalfi — APPROVED** [review]
One small thing, but LGTM

**Comment by rudygalfi** [comment]
I think @cramforce needs to merge.

File: README.md (hunk 1 above)
Comment (ghost) [reply README.md]: fixed

**Pull request merged**